### ISSUE TITLE
Docker Build Gets Stuck, Fails

### ISSUE THREAD

**Issue by agroce** [trailofbits/deepstate#225]
```creating 'dist/deepstate-0.1-py3.6.egg' and adding 'build/bdist.linux-x86_64/egg' to it
removing 'build/bdist.linux-x86_64/egg' (and everything under it)
Processing deepstate-0.1-py3.6.egg
removing '/usr/local/lib/python3.6/dist-packages/deepstate-0.1-py3.6.egg' (and everything under it)
creating /usr/local/lib/python3.6/dist-packages/deepstate-0.1-py3.6.egg
Extracting deepstate-0.1-py3.6.egg to /usr/local/lib/python3.6/dist-packages
deepstate 0.1 is already the active version in easy-install.pth
Installing deepstate script to /usr/local/bin
Installing deepstate-afl script to /usr/local/bin
Installing deepstate-angora script to /usr/local/bin
Installing deepstate-angr script to /usr/local/bin
Installing deepstate-eclipser script to /usr/local/bin
Installing deepstate-manticore script to /usr/local/bin
Installing deepstate-reduce script to /usr/local/bin

Installed /usr/local/lib/python3.6/dist-packages/deepstate-0.1-py3.6.egg
Processing dependencies for deepstate==0.1
error: z3-solver 4.8.5.0 is installed but z3-solver==4.5.1.0.post2 is required by {'claripy'}```

**Comment by agroce** [comment]
After the z3 error, nothing happens, forever...

**Comment by agroce** [comment]
Confirmed in Travis and multiple local setups, mac OS and linux too

**Comment by agroce** [comment]
For now this is not a problem, but only because we removed Angora from the Docker build

**Comment by GrosQuildu** [comment]
Fixed in #315, #326 and some others. Closing.
